### PR TITLE
Added multiple lights to TP-Link lights

### PR DESCRIPTION
**Pull request by cooperj**
Added the multiple lights text same as TP-Link switches.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/light.tplink.markdown [hunk 1]
```diff
@@ -32,6 +32,19 @@ light:
     host: IP_ADDRESS
 ```
 
+Adding multiple lights:
+
+You may need to add [multiple lights](https://community.home-assistant.io/t/multiple-tp-link-switches/6935) and the config would need to include multiple lights separately.
+
+```yaml
+# Example configuration.yaml entry
+switch:
```
Comment (rohankapoorcom) [inline source/_components/light.tplink.markdown]: This should be a `light` right?

Comment (rohankapoorcom) [inline source/_components/light.tplink.markdown]: This should be a `light` right?

**Review by rohankapoorcom — CHANGES_REQUESTED** [review]

File: source/_components/light.tplink.markdown (hunk 1 above)
Comment (cooperj) [reply source/_components/light.tplink.markdown]: yes it should... ty

**Comment by frenck** [comment]
Rebased PR onto `current`.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @cooperj! 

Happy Hacktoberfest 🎉

**Pull request merged**